docs/ballerina-by-example/examples/constants/constants.description

```diff
@@ -1 +1,2 @@
-// Ballerina supports value typed constants except for the blob type.
+// Constants are declared just like variables, but with the final keyword.
```

Comment:
final is not a keyword. It should be "@final annotation"

Reply:
Fixed this and sent an update to the PR.